specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/JobPrivateEndpoints.json

```diff
@@ -0,0 +1,329 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-05-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/jobAgents/{jobAgentName}/privateEndpoints": {
+      "get": {
+        "tags": [
+          "JobPrivateEndpoints"
+        ],
+        "description": "Gets a list of job agent private endpoints.",
+        "operationId": "JobPrivateEndpoints_ListByAgent",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceGroupParameter"
```

Comment:
>v1 [](http://example.com/codeflow?start=37&length=2)

Use latest (v3 or v5) common types, don't use v1 types.

Reply:
Hmm, I can't find an example of other specs under .\specification\sql\resource-manager\Microsoft.Sql\ using v3 or v5 types. I don't think this is something I can control.